src/test/java/testsuite/integration/container/HikariCPIntegrationTest.java

```diff
@@ -0,0 +1,170 @@
+package testsuite.integration.container;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+import org.junit.jupiter.api.*;
```

Comment:
Import classes instead of `*`. IntelliJ's optimize imports should help you with that